packages/nextjs/utils/scaffold-eth/networks.ts

```diff
@@ -5,6 +5,7 @@ import scaffoldConfig from "~~/scaffold.config";
 export type TChainAttributes = {
   // color | [lightThemeColor, darkThemeColor]
   color: string | [string, string];
+  tokenAddress?: string;
```

Comment:
Nitpick : Should we change it to `ethMainnetTokenAddress` ? 

Lol I am bad at names, but current one might be kind of confusing

Reply:
`nativeCurrencyTokenAddress`? IDK, I suck at naming too haha

Reply:
`nativeCurrencyTokenAddress ` seems more intuitive  :), also I think we should add a small comment above it that help developers know why they need to add it maybe : 

"Used to show current native currency price" (can be improved a lot 😅)

Reply:
I do agree we should add a comment explaining the variable so that developers can use it for other chains if required.

And how about `nativeCurrencyMainnetAddress` for the variable name since it would be more precise explaining that its the chain's native currencies mainnet address.

Reply:
> I think `nativeCurrencyTokenAddress ` is suffice, the word `Token` adds some value 😆 but just my personal preference also we will be adding a comment about the above so will be easy to understand 🙌

ya fair, works